src/routes/projects.routes.js

```diff
@@ -216,46 +216,50 @@ router.get(
  * @apiParam {String} version the version of the project
  * @apiParam {String} license the name of the license
  * @apiParam {[String]} authors a string of comma separated values
- * @apiParam {String} containerizationTool the name of the containerization tool or <None>
- * @apiParam {String} continuousIntegrationTool the name of the continuous integration tool or
- * <None>
- * @apiParam {String} deploymentTool the name of the deployment tool or <None>
- * @apiParam {String} webFramework the name of the web framework or <None>
+ * @apiParam {String} containerizationTool the name of the containerization tool
+ * @apiParam {String} continuousIntegrationTool the name of the continuous integration tool
+ * @apiParam {String} deploymentTool the name of the deployment tool
+ * @apiParam {String} webFramework the name of the web framework
  *
  * @apiParamExample {json} Request Example:
  * {
- *  "projectName": "hammer-io",
- *  "description": "Hit it with a Hammer!",
- *  "version": "0.0.1",
- *  "license": "MIT",
- *  "authors": "Holmgang, Jack",
- *  "containerizationTool": "2",
- *  "continuousIntegrationTool": "1",
- *  "deploymentTool": "3",
- *  "webFramework": "4"
+ *     "projectName": "hammer-io",
+ *     "description": "Hit it with a hammer!",
+ *     "isPrivateProject": false,
+ *     "version": "0.0.0",
+ *     "author": "Jack",
+ *     "license": "MIT",
+ *     "sourceControl": "0271d7cf-8a2a-4ebb-9bc0-bd1278a4c3b8",
+ *     "ci": "9c9cace2-995b-462b-848b-48c47c5f31d3",
+ *     "containerization": "d0db39f6-452e-4461-abf5-146e5d8e4415",
+ *     "deployment": "1d56c031-2a60-4d61-a7a7-a260e7f98610",
+ *     "web": "7f6f84c3-3123-40ef-935b-479a133e7d63",
+ *     "test": "4a6c48ce-a3ef-4f7c-bb13-bef287dc6daa"
```

Comment:
generalize the API documentation

Reply:
Those are my local keys, but won't be on any one else's, but I wanted to show an example of the ids.  Otherwise I could use a random string.

Reply:
I think the ids are fine, gives a better idea of what the fields look like